ouroboros-consensus/test-util/Test/Util/TestBlock.hs

```diff
@@ -114,18 +117,21 @@ import qualified Ouroboros.Consensus.Util.SlotBounded as SB
 -- in-memory representation).
 --
 -- The 'BlockNo' of the corresponding block is just the length of the list.
-newtype TestHash = TestHash {
+newtype TestHash = UnsafeTestHash {
```

Comment:
The same pattern synonym trick can be used here.

Reply:
Done.